### PR TITLE
OSDOCS-10595#removing OS Online reference

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions:
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-10595

Link to docs preview:
https://83382--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/overview/installing-preparing.html#installing-preparing-install-manage 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
@yasun1, PTAL

File: installing/overview/installing-preparing.adoc [hunk 1]
```diff
@@ -39,7 +39,7 @@ endif::openshift-origin[]
...
+If you want to use {product-title} but you do not want to manage the cluster yourself, you can choose from several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated]. You can also use OpenShift as a managed service on {azure-short}, {aws-short}, {ibm-cloud-name}, or {gcp-full}. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page. If you install an {product-title} cluster with a cloud virtual machine as a virtual bare metal, the corresponding cloud-based storage is not supported.
```
Comment (ShaunaDiaz) [inline installing/overview/installing-preparing.adoc:42]:
```suggestion
If you want to use {product-title} but you do not want to manage the cluster yourself, you can choose from several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated]. You can also use OpenShift as a managed service on {azure-short}, {aws-short}, {ibm-cloud-name}, or {gcp-full}. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page. If you install an {product-title} cluster with a cloud virtual machine as a virtual bare metal machine, the corresponding cloud-based storage is not supported.
```
out of scope, but wonder if "machine" should be added here

**Review by ShaunaDiaz — COMMENTED** [review]
one comment on content that is already there; up to you
otherwise LGTM!

File: installing/overview/installing-preparing.adoc (hunk 1 above)
Comment (brendan-daly-red-hat) [reply installing/overview/installing-preparing.adoc:42]: Thanks. 'bare metal' seems to be the convention (e.g., https://docs.openshift.com/container-platform/4.17/installing/overview/installing-preparing.html#installing-preparing-security), so I don't think 'machine' is needed here.

**Review by yasun1 — COMMENTED** [review]
/lgtm

File: installing/overview/installing-preparing.adoc (hunk 1 above)
Comment (yasun1) [reply installing/overview/installing-preparing.adoc:42]: +1

**Comment by stevsmit** [comment]
lgtm.

**Pull request merged**